projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,1554 @@
+	"tag.item.quark.parrot_feed": "鹦鹉的食物",
+	"tag.item.quark.pipes": "管道",
+	"tag.item.quark.posts": "木杆",
+	"tag.item.quark.revertable_chests": "可以转换成原版箱子的箱子",
+	"tag.item.quark.revertable_trapped_chests": "可以转换成原版陷阱箱的陷阱箱",
+	"tag.item.quark.runes": "符文",
+	"tag.item.quark.runes_lootable": "可以作为战利品获得的符文",
+	"tag.item.quark.seed_pouch_holdable": "种子袋可以容纳的物品",
+	"tag.item.quark.shards": "玻璃碎片",
+	"tag.item.quark.stained_framed_glass_panes": "染色镶框玻璃板",
+	"tag.item.quark.stained_framed_glasses": "染色镶框玻璃",
+	"tag.item.quark.stone_tool_materials": "石质工具材料",
+	"tag.item.quark.stools": "凳子",
+	"tag.item.quark.vertical_slabs": "竖直台阶",
+	"tag.item.quark.wooden_vertical_slabs": "木质竖直台阶",
+
+	"advancements.quark.influence.title": "潜移默化",
+	"advancements.quark.influence.description": "矩阵附魔时，使用蜡烛来影响特定魔咒的产生几率",
+	"advancements.quark.pat_potato.title": "大土豆，小土豆",
+	"advancements.quark.pat_potato.description": "爱抚一只小土豆",
+	"advancements.quark.watcher_center.title": "观察者由谁观察？",
+	"advancements.quark.watcher_center.description": "看向末影观察者的面心",
+	"advancements.quark.crawl_in_hollow_log.title": "在树皮中爬行",
+	"advancements.quark.crawl_in_hollow_log.description": "在空心原木中爬行",
+	"advancements.quark.get_forgotten_hat.title": "博物馆的老古董",
+	"advancements.quark.get_forgotten_hat.description": "在地下击杀一名遗忘者并缴获其帽子",
+	"advancements.quark.get_diamond_heart.title": "Shape of My Heart",
+	"advancements.quark.get_diamond_heart.description": "在地下潜行靠近石精灵并击碎它的外壳以获得钻石之心",
+	"advancements.quark.make_stoneling.title": "年轻的逐梦人",
+	"advancements.quark.make_stoneling.description": "通过对任意石头使用石精灵的钻石之心来给它一具崭新的躯体",
+	"advancements.quark.mine_toretoise.title": "矿石龟！",
+	"advancements.quark.mine_toretoise.description": "用镐子挖取野生矿石陆龟背上的矿物",
+	"advancements.quark.mine_fed_toretoise.title": "我们发财了！",
+	"advancements.quark.mine_fed_toretoise.description": "给矿石陆龟喂食发光浆果使其矿物重新生成，之后再次收获其背上的矿物",
+	"advancements.quark.throw_pickarang.title": "不知怎么的就回来了",
+	"advancements.quark.throw_pickarang.description": "掷出一枚皮卡镖",
+	"advancements.quark.retreive_flamerang.title": "Fly to the Sky, Fire Bird",
+	"advancements.quark.retreive_flamerang.description": "掷出一枚界炎镖，在其燃烧时取回一个物品",
+	"advancements.quark.overlevel_enchant.title": "超越极限",
+	"advancements.quark.overlevel_enchant.description": "使用古卷来超越魔咒的最高等级",
+	"advancements.quark.instamine_deepslate.title": "超越瞬间的极限",
+	"advancements.quark.instamine_deepslate.description": "同时使用效率VI和急迫II来瞬间破坏深板岩",
+	"advancements.quark.redirect_beacon.title": "See The Lights",
+	"advancements.quark.redirect_beacon.description": "使用刚玉簇改变信标光柱的路径",
+	"advancements.quark.get_bottled_cloud.title": "云存储",
+	"advancements.quark.get_bottled_cloud.description": "把云朵捕捉到瓶子中",
+	"advancements.quark.apply_color_rune.title": "雄伟者之矜持",
+	"advancements.quark.apply_color_rune.description": "使用铁砧将彩色符文应用到物品上",
+	"advancements.quark.wear_full_rainbow.title": "吃定彩虹",
+	"advancements.quark.wear_full_rainbow.description": "装备一身带有彩虹符文效果的盔甲",
+	"advancements.quark.throw_parrot_egg.title": "只是鹦鹉",
```

```suggestion
	"advancements.quark.throw_parrot_egg.title": "你哪也不是",
```
“Just the Parrots”进度我认为是在对应原版的“The Parrots and the Bats”（我从哪里来？），所以我的方案做了一点额外发挥。

---

这样译是错误的，因为 the birds and the bees 是以性教育的图像代指性教育本身。
若认为 Just the Parrots 是 The Parrots and the Bats 的半句的话，也应加入性教育的意思，比如鸟下蛋繁衍之类的。
建议译作“你是蛋里面出来的”

---

那简化为“你从蛋里来”呢？正好和“我从哪里来？”对应。

---

随便，你/我也随便